src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+  
+        <label for="surname">Surname:</label>
+        <input
+        class="input"
```

Comment:
Please add two spaces. Please fix for all cases!
![image](https://user-images.githubusercontent.com/54936044/224478855-15ce2943-a728-4841-b69f-48b862f053c8.png)